Database/Corrections/Wotlk/wotlkQuestFixes.lua

```diff
@@ -1036,6 +1036,9 @@ function QuestieWotlkQuestFixes:Load()
             [questKeys.name] = "Hodir's Call",
             [questKeys.objectives] = {{{29974,"Niffelem Forefather freed"},},nil,nil,nil,{{{30144,30135},30144,},},},
         },
+        [12979] = {
+			[questKeys.objectives] = {nil,nil,{{42204},},nil,nil,},
```

Comment:
This is fine, but just for future reference you don't need to `nil` the rest of the table *after* your data. The preceding `nil`s are to ensure your data goes into the 3rd parameter (itemObjective), anything after that is needless (but legal) unless you need to add data to a later parameter.

The very act of re-defining the table will wipe out any existing data in `questKeys.objectives`.

Reply:
Gotcha, thanks.